### PR TITLE
list and map

### PR DESCRIPTION
**Pull request by anniethewebcoder**
here is my code.

### CODE REVIEW

File: src/components/Activities/Activities.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        <div key={item.id}>
+          <h2>{item.name}</h2>
+          <p>{item.category}</p>
+          <p>{item.description}</p>
+          <p>{item.voteCount}</p>
+        </div>
+      ))}
+    </div>
+  );
+}
```
Comment (angelkbrown) [inline src/components/Activities/Activities.tsx]: Nice! I like that you followed typography conventions with the main header as `h1` and activity names as `h2`s! And I also like that you created a components directory! 👍

File: src/App.tsx [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
+    // <main className={styles.main}>
+    //   <img className={styles.logo} alt="React logo" width="400px" src={Logo} />
+    //   <DataContainer />
+    //   <div>
+    //     {`VITE_SOME_ENVIRONMENT_VARIABLE from .env.local: "${
+    //       import.meta.env.VITE_SOME_ENVIRONMENT_VARIABLE
+    //     }"`}
+    //   </div>
+    //   <HelloWorld msg="Hello EE Practicum Team 2!" />
+    // </main>
```
Comment (angelkbrown) [inline src/App.tsx]: Don't be afraid to delete! That's what's good about version control -- we can always look at past versions if we want to add something back in.

**Review by angelkbrown — APPROVED** [review]
Good job, Annie! 🎉 I will defer to Darga for any more nuanced feedback on implementation details, especially around typescript. But I ran your branch locally and it worked great for me! 👍

File: src/App.tsx (hunk 2 above)
Comment (dargaCode) [reply src/App.tsx]: 💯

File: src/components/Activities/Activities.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+
+export default function Activities() {
```
Comment (dargaCode) [inline src/components/Activities/Activities.tsx]: Please rename the file, the folder, and the component to `ActivitiesList`

Sorry I forgot to note this in the ticket!

File: src/components/Activities/Activities.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+      {list.map((item) => (
+        <div key={item.id}>
+          <h2>{item.name}</h2>
+          <p>{item.category}</p>
+          <p>{item.description}</p>
+          <p>{item.voteCount}</p>
+        </div>
+      ))}
+    </div>
+  );
```
Comment (dargaCode) [inline src/components/Activities/Activities.tsx]: looking good!

File: src/components/Activities/Activities.tsx (hunk 1 above)
Comment (dargaCode) [reply src/components/Activities/Activities.tsx]: Yes great job!

File: src/components/Activities/Activities.tsx [hunk 5]
```diff
@@ -0,0 +1,73 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+
+export default function Activities() {
+  const list = [
```
Comment (dargaCode) [inline src/components/Activities/Activities.tsx]: Eventually we're going to need this component to accept props. Can you please move this activities array out to `App` and then pass it into `ActivitiesList` as a prop called `activities`?

**Review by dargaCode — CHANGES_REQUESTED** [review]
Great job Annie! 

I'm blindsiding you here with a couple of specific requirements I forgot to add to the ticket, sorry about that! Please make these changes and then you can "re-request" review from me in the upper right corner 🙏

**Comment by dargaCode** [comment]
ps, please link the PR to the ticket it's resolving 🔗

File: src/App.tsx [hunk 6]
```diff
@@ -1,20 +1,24 @@
...
-    <main className={styles.main}>
-      <img className={styles.logo} alt="React logo" width="400px" src={Logo} />
-      <DataContainer />
-      <div>
-        {`VITE_SOME_ENVIRONMENT_VARIABLE from .env.local: "${
-          import.meta.env.VITE_SOME_ENVIRONMENT_VARIABLE
-        }"`}
-      </div>
-      <HelloWorld msg="Hello EE Practicum Team 2!" />
-    </main>
```
Comment (dargaCode) [inline src/App.tsx]: For now please leave the old code in App as well, and add `ActivitiesList` to the render right below `HelloWorld`

(We can come back later on and remove these other components)

**Comment by anniethewebcoder** [comment]
Okay. I moved the list to app.tsx and passed the prop. I did the basic CSS to get started.

File: src/components/Activities/ActivitiesList.tsx
Comment (dargaCode) [inline src/components/Activities/ActivitiesList.tsx:1]: Please rename the folder and the component name to match the `ActivitiesList` pattern too 🙏

File: src/components/Activities/ActivitiesList.tsx [hunk 7]
```diff
@@ -0,0 +1,24 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+
+import styles from "./ActivitiesList.module.css";
+
+interface ActivitiesListProps {
+  list: Array<any>;
```
Comment (dargaCode) [inline src/components/Activities/ActivitiesList.tsx:7]: Please rename `list` prop to `activities`, so it's more descriptive

**Review by dargaCode — CHANGES_REQUESTED** [review]
Thanks Annie! Great job adding Typescript here 💪 

Still a couple of naming inconsistencies and then we'll be good to go!

**Pull request merged**